calico_containers/libnetwork_plugin/datastore_libnetwork.py

```diff
@@ -0,0 +1,35 @@
+from etcd import EtcdKeyNotFound
+import json
+from pycalico.ipam import IPAMClient
+
+PREFIX = "/CALICO_DOCKER/"
```

Comment:
I realise this is temporary, but would still be nice to make it something more like /calico/docker/v1